src/bin/lpython.cpp

```diff
@@ -1244,11 +1244,17 @@ int link_executable(const std::vector<std::string> &infiles,
             cmd += s + " ";
         }
         cmd += " -I " + rtlib_header_dir;
+        if (compiler_options.enable_symengine) {
+            cmd += " -I${CONDA_PREFIX}/include";
+        }
         cmd += " -L" + base_path
             + " -Wl,-rpath," + base_path + " -l" + runtime_lib + " -lm";
+        if (compiler_options.enable_symengine) {
+            cmd += " -L$CONDA_PREFIX/lib -Wl,-rpath, $CONDA_PREFIX/lib -lsymengine";
+        }
         if (compiler_options.enable_cpython) {
             std::string py_version = "3.10";
-            std::string py_flags = R"(-I $CONDA_PREFIX/include/python)" + py_version + R"( -L$CONDA_PREFIX/lib -Wl,-rpath -Wl,$CONDA_PREFIX/lib -lpython)" + py_version + R"()";
+            std::string py_flags = R"(-I $CONDA_PREFIX/include/python)" + py_version + R"( -L$CONDA_PREFIX/lib -Wl,-rpath, $CONDA_PREFIX/lib -lpython)" + py_version + R"()";
```

Comment:
I am not sure about this change. I thought the correct syntax is `-Wl,-rpath -Wl,$CONDA_PREFIX/lib`. Does it not work? I know it's slightly different on macOS and Linux, and one has to be careful here.

Reply:
Yeah even I thought the correct syntax involves having `-Wl` after `-rpath` . Not sure if this is affecting the CI !

Reply:
@anutosh491 I think both the syntaxes (`-Wl,a,b,c` and `-Wl,a -Wl,b -Wl,c`) would/should work.